public/chat/santaChat.js

```diff
@@ -31,11 +34,10 @@ $(async function() {
         $(chatElement).find('[data-name="chatDate"]').text(dateStr);
         $(chatElement).find('[data-name="chatFrom"]').text(`From: ${item.from || 'Anonymous'}`);
 
-        $(chatElement).find('button').on('click', function() {
-          $.post(`${apiUrl}/delete`, { _id: item._id }, result => {
-            if (result.success) $(chatElement).remove();
-            showAlert(result);
-          });
+        $(chatElement).find('[data-name="msgDelete"]').on('click', function() {
+          $('#deletePairDialog').prop('hidden', false);
```

Comment:
what's this used for?

Reply:
With the help of the selector, we find the button with which we open the dialog

Reply:
I was referring to this line: $('#deletePairDialog').prop('hidden', false);